src/test/java/bio/terra/service/filedata/azure/AzureSynapsePdaoConnectedTest.java

```diff
@@ -0,0 +1,167 @@
+    destinationDataSourceName = DESTINATION_DATA_SOURCE_PREFIX + randomFlightId;
+    tableName = TABLE_NAME_PREFIX + randomFlightId;
+
+    connectedOperations.stubOutSamCalls(samService);
+    billingProfile =
+        connectedOperations.createProfileForAccount(testConfig.getGoogleBillingAccountId());
+    // TODO - we don't really need a full dataset
+    // Replace this with just an entry in the db w/ the table schema
+    datasetSummary = connectedOperations.createDataset(billingProfile, "azure-simple-dataset.json");
+  }
+
+  @After
+  public void cleanup() throws Exception {
+    azureSynapsePdao.cleanSynapseEntries(
+        Arrays.asList(tableName),
+        Arrays.asList(destinationDataSourceName, controlFileDataSourceName),
+        Arrays.asList(destinationScopedCredentialName, controlFileScopedCredentialName));
+
+    // TODO - Clean out test parquet files
+
+    connectedOperations.teardown();
+  }
+
+  @Test
+  public void testSynapseQueryCSV() throws Exception {
+    IngestRequestModel ingestRequestModel =
+        new IngestRequestModel().format(FormatEnum.CSV).csvSkipLeadingRows(2);
+    String ingestFileName = "azure-simple-dataset-ingest-request.csv";
+    testSynapseQuery(ingestRequestModel, ingestFileName);
+  }
+
+  @Test
+  public void testSynapseQueryJSON() throws Exception {
+    IngestRequestModel ingestRequestModel = new IngestRequestModel().format(FormatEnum.JSON);
+    String ingestFileName = "azure-simple-dataset-ingest-request.json";
+    testSynapseQuery(ingestRequestModel, ingestFileName);
+  }
+
+  private void testSynapseQuery(IngestRequestModel ingestRequestModel, String ingestFileName)
+      throws Exception {
+    // Currently, the parquet files will live in the same location as the ingest control file
+    String destinationTableName = "participant";
+    String destinationParquetFile = "parquet/" + randomFlightId + ".parquet";
+
+    UUID tenantId = testConfig.getTargetTenantId();
+
+    // ---- ingest steps ---
+    // 1 - Create external data source for the ingest control file
+    String ingestFileLocation = "https://tdrconnectedsrc1.blob.core.windows.net/synapsetestdata";
+
+    azureSynapsePdao.createExternalDataSource(
+        ingestFileLocation,
+        tenantId,
+        controlFileScopedCredentialName,
+        controlFileDataSourceName,
+        SASPermission.READ_ONLY);
+
+    // 2 - Create the external data source for the destination
+    // where we'll write the resulting parquet files
+    String parquetDestinationLocation =
+        "https://tdrconnectedsrc1.blob.core.windows.net/synapsetestdata";
+    azureSynapsePdao.createExternalDataSource(
+        parquetDestinationLocation,
+        tenantId,
+        destinationScopedCredentialName,
+        destinationDataSourceName,
+        SASPermission.WRITE_PARQUET);
+
+    // TODO - Add basic check to make sure the data source is created successfully
+    // Maybe a basic query?
```

Comment:
Yeah, that's not a bad idea

Reply:
I ended up not doing this - The next request ends up checking if it's successfully created. But, I did add a check that reads the parquet file and validates that it's correct.